using FullCourse.exceptions;

namespace FullCourse;

internal class MainClass
{

    private static void Main()
    {
        RandomNumber.Guess();
        Division.main();
    }
}